x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -126,12 +126,18 @@ export const ActionForm = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const isCaseConnector = (connector: ActionConnector) =>
+    !connector.isPreconfigured &&
```

Comment:
I did `!connector.isPreconfigured` to make typescript happy. The type of ActionConnector is 

```
type ActionConnector<Config = Record<string, any>, Secrets = Record<string, any>> =
  | PreConfiguredActionConnector
  | UserConfiguredActionConnector<Config, Secrets>;
```

Type `PreConfiguredActionConnector` does not have the `config` attribute. Is it possible for a user to create a preconfigured connector with `config`?

Reply:
Users shouldn't be able to create preconfigured connectors or to change its configuration (and secrets), so that's why this type PreConfiguredActionConnector was created.

Reply:
I see thanks a lot!